### PR TITLE
Documentation for Powerstore Install support in csm operator

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Added documentation for Powerstore install for CSM Operator

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/613 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csmoperator/drivers/powerstore.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+| ***Common parameters for node and controller*** |
+| X_CSI_POWERSTORE_NODE_NAME_PREFIX | Prefix to add to each node registered by the CSI driver | Yes | "csi-node" 
+| X_CSI_FC_PORTS_FILTER_FILE_PATH | To set path to the file which provides a list of WWPN which should be used by the driver for FC connection on this node | No | "/etc/fc-ports-filter" |
+| ***Controller parameters*** |
+| X_CSI_POWERSTORE_EXTERNAL_ACCESS | allows specifying additional entries for hostAccess of NFS volumes. Both single IP address and subnet are valid entries | No | " "|
+| X_CSI_NFS_ACLS | Defines permissions - POSIX mode bits or NFSv4 ACLs, to be set on NFS target mount directory. | No | "0777" |
+| ***Node parameters*** |
+| X_CSI_POWERSTORE_ENABLE_CHAP | Set to true if you want to enable iSCSI CHAP feature | No | false |
+
+4.  Execute the following command to create PowerStore custom resource:`kubectl create -f <input_sample_file.yaml>`. The above command will deploy the CSI PowerStore driver in the namespace specified in the input YAML file
```
Comment (rsedlock1958) [inline content/docs/deployment/csmoperator/drivers/powerstore.md:87]: Change "The above" to "This"

File: content/docs/deployment/csmoperator/drivers/powerstore.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      - After that the driver should be installed, you can check the condition of driver pods by running `kubectl get all -n <driver-namespace>`
```
Comment (rsedlock1958) [inline content/docs/deployment/csmoperator/drivers/powerstore.md:88]: Change "After that" to "Next,"

File: content/docs/deployment/csmoperator/drivers/powerstore.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+5.  [Verify the CSI Driver installation](../#verifying-the-driver-installation)
+    
+**Note** : 
+   1. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
+   2. Also, snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation.
```
Comment (rsedlock1958) [inline content/docs/deployment/csmoperator/drivers/powerstore.md:94]: Delete "Also,". Initial uppercase on Snapshotter. Delete "to choose," Add full stop ("." period) after "optional". Replace "it" with "They". Replace "it comes" with "are defaults".

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Several edits are requiired.

File: content/docs/deployment/csmoperator/drivers/powerstore.md (hunk 1 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/powerstore.md]: Done

File: content/docs/deployment/csmoperator/drivers/powerstore.md (hunk 2 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/powerstore.md]: Done

File: content/docs/deployment/csmoperator/drivers/powerstore.md (hunk 3 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/powerstore.md]: Done

**Review by rensyct — DISMISSED** [review]

**Review by karthikk92 — DISMISSED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**